packages/runtime/src/index.ts

```diff
@@ -251,13 +267,14 @@ export default function init(inlineConfig: RuntimeOptions = {}) {
   function observe() {
     if (observing)
       return
-    const target = html() || defaultDocument.body
+    const target = runtimeOptions.observer?.target ? runtimeOptions.observer.target() : (html() || defaultDocument.body)
     if (!target)
       return
     mutationObserver.observe(target, {
       childList: true,
       subtree: true,
       attributes: true,
+      ...(runtimeOptions.observer?.attributeFilter ? { attributeFilter: runtimeOptions.observer.attributeFilter } : {}),
```

Comment:
I think we can directly use `imeOptions.observer?.attributeFilter` since `attributeFilter` is optional by default ?

Reply:
It is optional, but `runtimeOptions.observer?.attributeFilter` will be undefined, and the MutationObserver options are an object which we are extending if required, afaik I can't just "spread" undefined, or maybe I'm not understanding correctly what you mean?

Reply:
@ydcjeff I think that's indeed what you meant, I tried and it works, if `attributeFilter: undefined` it monitors all attributes, simplified the code per your suggestion! Thanks for checking and giving great feedback, reduced bytes added to bundle!